packages/amazonq/.changes/next-release/Feature-69f0d3bf-3e57-4ad6-93ea-48de3c890331.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Add LSP for Amazon Q Chat Local Indexing"
```

Comment:
The changelog is a "mini blog" for advertising new features. It also allows markdown, so you can link to external URLs.

 Is there a way to describe this so that customers can discover the actual usefulness of the feature?

Reply:
Added more description in the change log.